app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -39,6 +39,43 @@ def info
         render layout: !request.xhr?
       end
 
+      def schedule_publish_new
+        render layout: !request.xhr?
+      end
+
+      def schedule_publish
+        delete_current_scheduled_job(@page.job_id) if @page.job_id
+        job_id = ScheduleCmsPagePublish.perform_at(time_to_publish_utc, @page.id)
+        @page.update_attributes(job_id: job_id, scheduled_publish_time: time_to_publish_pst) 
+        flash[:notice] = "#{@page.name} scheduled to publish at #{time_to_publish_pst}PT"
+        redirect_to admin_pages_path
+      end
+
+      def time_to_publish_pst
+        year = params[:scheduled_publish]['time_to_publish(1i)'].to_i
+        month = params[:scheduled_publish]['time_to_publish(2i)'].to_i
+        day = params[:scheduled_publish]['time_to_publish(3i)'].to_i
+        hours = params[:scheduled_publish]['time_to_publish(4i)'].to_i
+        minutes = params[:scheduled_publish]['time_to_publish(5i)'].to_i
+        @time_to_publish_pst ||= Time.zone.parse("#{year}-#{month}-#{day} #{hours}:#{minutes}")
```

Comment:
Do you want to be explicit about the timezone here? For critical calculations (like 'go live at midnight') Is it enough to rely on the system default which might change and throw things out of whack? (I just don't know.)

I know the calculations for rebill dates are all based on ET/EST not PT/PST